lib/parzival/gamification/mission.ex

```diff
@@ -40,7 +40,7 @@ defmodule Parzival.Gamification.Mission do
 
     belongs_to :difficulty, Difficulty
 
-    has_many :tasks, Task, on_replace: :delete_if_exists
+    has_many :tasks, Task
```

Comment:
Undo this change. Otherwise, it would not be possible to see a mission's tasks Or even to see who completed them.